build.sh

```diff
@@ -20,8 +20,10 @@ if [ "$1" == "dev-build" ];then
 
     # remove coreboot path
     shift
-    docker run --rm -it -v $cb_path:/home/coreboot/coreboot -v $PWD/scripts:/home/coreboot/scripts pcengines/pce-fw-builder:latest \
+    docker run --rm -it -v $cb_path:/home/coreboot/coreboot  \
+        -v $PWD/scripts:/home/coreboot/scripts pcengines/pce-fw-builder:latest \
         /home/coreboot/scripts/pce-fw-builder.sh $*
+
 elif [ "$1" == "release" ]; then
     if [ -d release ]; then
         sudo rm -rf release
```

Comment:
@pietrushnic I wonder how sudo will behave on CI.

Reply:
@miczyg1 are we running root on CI?

Reply:
@pietrushnic we are running default user called coreboot